### DISCUSSION TITLE
Ingress/Egress Gateway redundancy

### DISCUSSION THREAD

**Discussion by SebastienDorgan** [gravitl/netmaker#1383] [Q&A]
Hi,
I would like to create a Kubernetes HA cluster distributed across 3 VPCs in 3 different clouds: C1/VPC1, C2/VPC2, C3/VPC3.
The 3 masters would be deployed in VPC1. 
This is a fairly large cluster with several dozen workers in each VPC.
To avoid creating a full-mesh, I was planning to use Ingress/Egress gateways at the edge of VPC2 and VPC3.
Is it possible to create a redundancy to make the connection between VPC1 masters and VPC2 et VPC3 workers highly available?

**Comment by afeiszli** [comment]
Not currently, no. There is no HA for Egress/Ingress.
